--- a/chrome/browser/ui/BUILD.gn
+++ b/chrome/browser/ui/BUILD.gn
@@ -398,8 +398,12 @@
 # circular includes from each other.
 static_library("ui") {
   sources = [
+    "//samo/shell/samo_launcher_host.cc",
+    "//samo/shell/samo_launcher_host.h",
     "//samo/shell/samo_shell_view.cc",
     "//samo/shell/samo_shell_view.h",
+    "//samo/webui/samo_ui_handler.cc",
+    "//samo/webui/samo_ui_handler.h",
     "crypto_module_password_dialog.h",
     "idle_bubble.h",
     "managed_ui.cc",
